src/GovUk.Education.ExploreEducationStatistics.Data.Model/LocationAttribute.cs

```diff
@@ -182,6 +196,7 @@ public record PlanningArea(string? Code, string? Name) : LocationAttribute(Code,
 
 public record Provider(string? Code, string? Name) : LocationAttribute(Code, Name)
 {
+    [SuppressMessage("ReSharper", "StringLiteralTypo")]
```

Comment:
Perhaps we could exempt the `.idea/dictionaries` folder from the gitignore, and commit a project-level shared dictionary to avoid having to suppress typos like this?

Reply:
Awesome stuff! 
Hrm that's a shame it wasn't as straight forward as I'd hoped, but it sounds like the Dotsettings file is a better way to go anyway :)